kuksa-client/kuksa_client/__main__.py

```diff
@@ -559,10 +559,15 @@ def do_disconnect(self, _args):
                 self.commThread.stop()
                 self.commThread = None
 
-    def checkConnection(self):
-        if self.commThread is None or not self.commThread.checkConnection():
+    def connection_established(self) -> bool:
```

Comment:
shouldn't we name this different. I see that we had aslo two checkConnection functions but I kind of find it confusing.

Reply:
Ping @lukasmittag